### PR TITLE
Feature/display trips

### LINKED ISSUE #3: A user should see all trips on the dashboard (past, present, upcoming and pending)

### LINKED ISSUE #6: User should see how much they have spent on trips in their dashboard
This should be calculated from the trips data and include a travel agent’s 10% fee

### PR DESCRIPTION
**Pull request by mcraig2342**
## Description
resolves #3, resolves #6
Please include a summary of the change and which issue is fixed.

The names of the users trip destinations are now displayed in on of the columns depending on trip date, the amount that a 
user has spent on trips (including pending trips) are now displayed on the dash

Please include which files need to be reviewed.
index.js,  dom-updates

screenshot

Screen Shot 2021-04-22 at 6 30 45 PM

### CODE REVIEW

File: src/dom-updates.js [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+  },
+
+  displayUserTrips(user) {
+    let todaysDate = "2020/3/28"
+    const pastTripList = document.getElementById('pastTripList')
+    const upcomingTripList = document.getElementById('upcomingTripList')
+    const ongoingTripList = document.getElementById('ongoingTripList')
+    const pendingTripList = document.getElementById('pendingTripList')
+
+    user.trips.forEach((trip, i) => {
```
Comment (RMartin0717) [inline src/dom-updates.js:19]: Like the way you organized the trips and kept it really straightforward and organized

File: src/dom-updates.js [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+      if (trip.status === 'pending') {
```
Comment (RMartin0717) [inline src/dom-updates.js:20]: I like this nested method to calculate the date. It's very well written in terms of developer empathy

**Review by RMartin0717 — APPROVED** [review]
Overall, it's looking really good and well organized

File: src/dom-updates.js [hunk 3]
```diff
@@ -1,5 +1,56 @@
...
+            `<p>${trip.correspondingDestination.destination}</p>`
+        } else {
+          pastTripList.innerHTML +=
+            `<p>${trip.correspondingDestination.destination}</p>`
+        }
+      }
+    });
+  },
+
+  calculateDate(trip, todaysDate) {
```
Comment (JSweet314) [inline src/dom-updates.js:38]: This is an interesting pattern. I wouldn't necessarily expect a function named `calculateDate` to return a string. Perhaps a change of name would make the intent of this function more clear.

File: src/dom-updates.js [hunk 4]
```diff
@@ -1,5 +1,56 @@
...
+    const upcomingTripList = document.getElementById('upcomingTripList')
+    const ongoingTripList = document.getElementById('ongoingTripList')
+    const pendingTripList = document.getElementById('pendingTripList')
+
+    user.trips.forEach((trip, i) => {
+      if (trip.status === 'pending') {
+        pendingTripList.innerHTML +=
+          `<p>${trip.correspondingDestination.destination}</p>`
+      } else {
+        if (this.calculateDate(trip, todaysDate) === 'ongoing') {
```
Comment (JSweet314) [inline src/dom-updates.js:24]: a switch statement might be more readable that this if-else block. Alternatively, capturing the result of the conditions in a variable ahead of the if/else blocks would improve readability as well.

File: src/index.html [hunk 5]
```diff
@@ -1,44 +1,56 @@
...
+  <nav>
+    <button type="button" class="dashboard-button">Dashboard</button>
+    <button type="button" class="book-trip-button">Book a trip</button>
+  </nav>
+  <main class="main-dashboard">
+    <section class="past-trips trip-box">
+      <header>
+        <h3 class="trip-header">Past trips</h3>
+      </header>
+      <ul id="pastTripList" class="trip-list">
```
Comment (JSweet314) [inline src/index.html:28]: any reason for using camelCase for id's and kebab-case for classes?

**Pull request merged**